web/components/issues/peek-overview/root.tsx

```diff
@@ -204,9 +208,9 @@ export const IssuePeekOverview: FC<IIssuePeekOverview> = observer((props) => {
       addIssueToCycle: async (workspaceSlug: string, projectId: string, cycleId: string, issueIds: string[]) => {
         try {
           await addIssueToCycle(workspaceSlug, projectId, cycleId, issueIds);
```

Comment:
Same goes here for cycle and module, since they have their own spinner see if you can remove the local spinner and use the loader